lib/msal-browser/src/config/Configuration.ts

```diff
@@ -236,7 +240,9 @@ export function buildConfiguration({ auth: userInputAuth, cache: userInputCache,
     const DEFAULT_CACHE_OPTIONS: Required<CacheOptions> = {
         cacheLocation: BrowserCacheLocation.SessionStorage,
         storeAuthStateInCookie: false,
-        secureCookies: false
+        secureCookies: false,
+        // Default cache migration to true if cache location is localStorage since entries are preserved across tabs/windows. Migration has little to no benefit in sessionStorage and memoryStorage
```

Comment:
We had discussed this morning that if using memory storage (session as well?) that we can just store objects and skip the serialization overhead there.  Can you file a task for this as a future optimization.

Reply:
Done. This would just be for memory storage. SessionStorage will still need to be serialized

Reply:
I was wondering if there is a way we can avoid de-serialization and serialization unless we reload the page. I think the tricky part is that the user can close the web page/reload at any point and we will lose the in-memory data structure if we do not do it per account creation and deletion. 

Are there any DOM APIs we can leverage? I do not think this is a blocker and may be the perf gains are not so significant. We can circle back on this.